clients/include/rocsolver_test.hpp

```
@@ -143,6 +143,14 @@ inline fs::path get_sparse_data_dir()
     fs::path p = fs::current_path();
     fs::path p_parent = p.parent_path();
     fs::path installed = p.root_directory() / "opt" / "rocm" / "share" / "rocsolver" / "test";
+    fs::path exe_relative = rocsolver_exepath() + "../" + "share/" + "rocsolver/" + "test/";
+    try
+    {
+        exe_relative = fs::canonical(exe_relative);
+    }
+    catch(const std::exception& ex)
+    {
+    }
```

The the generic `std::exception` and the empty catch block raises red flags to me, but I'm not entirely sure what we'd want to do about failure here anyway. Do we actually need to canonicalize this path? One way to have fewer errors to handle is to just do less stuff.